### DISCUSSION TITLE
Orientation of a talk bubble

### DISCUSSION THREAD

**Discussion by ax006ho** [phase1geo/Annotator#98] [Ideas]
Is there a way to make the top notch point to north instead to south? This would be very helpful if the talking person stays at the north border of a picture.

Any hint is appreciated!

**Comment by phase1geo** [comment]
Bubbles are not a released feature at this time.

**Reply by ax006ho** [reply]
Ok, I've installed Annotator from the github sources. I like that program very much and I hope, that my suggestion makes it into the next release.

**Comment by phase1geo** [comment]
Yes.  The plan is to allow the user to move the talking point to any position.

**Reply by ax006ho** [reply]
That's good to hear/read! Thank you for your courtesy!
